### PR TITLE
Fix example

### PR DESCRIPTION
**Pull request by sathvikbhagavan**

### CODE REVIEW

**Comment by sathvikbhagavan** [comment]
Also, the docs mentioned in the README should be updated as it points to an older version. 

But I don't see any CI for docs?

**Comment by ChrisRackauckas** [comment]
We should make this into a standard documenter documentation instead.

File: docs/src/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+-  Linear Interpolation
+-  Quadratic Interpolation
+-  Lagrange Interpolation
+-  Zero Spline
+-  Quadratic Spline
+-  Cubic Spline
+-  BSpline Global Curve Interpolation
+-  BSpline Global Curve Approximation
+-  Curve fitting
```
Comment (ChrisRackauckas) [inline docs/src/index.md:15]: Use the README? There's a lot more information that you can just copy-paste.

Comment (sathvikbhagavan) [reply docs/src/index.md]: Yes, done in [c75828a](https://github.com/PumasAI/DataInterpolations.jl/pull/173/commits/c75828a02ffe6c1d6be75f900dcb190031bf33d8)

**Comment by ChrisRackauckas** [comment]
While you're at it, can you add a section that documents the interface, i.e. how to interact with an interpolation `i(t)` but also calculate derivatives and integrals? Then I think we're good to go.

**Comment by ArnoStrouwen** [comment]
When this gets merged in. Does this go into the SciML docs -> dev tools -> 3rd party num utils?

**Comment by ChrisRackauckas** [comment]
I would treat it as non-third party. It's really weird it's not in SciML.

**Comment by sathvikbhagavan** [comment]
Any reason we can't move this to SciML?

**Comment by ChrisRackauckas** [comment]
It should.

**Review by ChrisRackauckas — APPROVED** [review]

**Pull request merged**